core/templates/dev/head/pages/exploration_editor/ExplorationSaveAndPublishButtonsDirective.js

```diff
@@ -118,10 +118,11 @@ oppia.directive('explorationSaveAndPublishButtons', [
               });
           };
 
-          $scope.showTooltip = function() {
-            if ($scope.isExplorationLockedForEditing() ||
+          $scope.showTooltipForDisabledButtons = function(buttonClass) {
+            if (
+              $scope.isExplorationLockedForEditing() ||
```

Comment:
Can you indent this line and the next by 2 spaces? This will separate it from the following code.

Reply:
It gives an indentation error: `Expected indentation of 14 spaces but found 16  indent`

Reply:
Ah ok. I wasn't sure if it would do that. Fine to leave as-is.